docs/testing.rst

```diff
@@ -381,6 +381,70 @@ functionality, the AioHTTPTestCase is provided::
    Handles executing an asynchronous function, using
    the :attr:`AioHTTPTestCase.loop` of the :class:`AioHTTPTestCase`.
 
+Patching unittest test cases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Patching test cases is tricky, when using python older than 3.8  :py:func:`unittest.mock.patch` does not behave as it has to.
+We recommend using :py:mod:`asynctest` that provides :py:func:`asynctest.patch` that is capable of creating
```

Comment:
How about
```suggestion
Patching test cases is tricky, when using python older than 3.8  :py:func:`~unittest.mock.patch` does not behave as it has to.
We recommend using :py:mod:`asynctest` that provides :py:func:`~asynctest.patch` that is capable of creating
```

Reply:
What is the difference?

Reply:
Never mind, I tried it, way better

Reply:
the anchor is rendered differently. `asynctest.patch` vs `patch`.

Reply:
Sorry I do not understand what are you pointing at. Do you mean that asynctest.patch does not have to be shortened?

Reply:
It was a reply to your earlier comment that I wrote before seeing your next comment. Nevermind.